docs/docs/optics.md

```diff
@@ -79,10 +79,11 @@ Following with the example above, if you need to read an enum value, you can do
 
 ```haskell
 {-# language MultiWayIf #-}
+{-# language OverloadedLabels #-}
 
-if | e `is` #SUNNY -> ...
```

Comment:
Please check that this actually type checks, I've written the code just in my head xD